### PR TITLE
Update misc.sp

### PR DESCRIPTION
**Pull request by Nubbeezez**
Edited BotQuota so that it doesnt fuck fps for client and fixes VAR spikes on server :)
Atleast made my personal server stop eating my fps
newb to sourcepawn/coding in total but trying to learn 👍 
Please give me criticism

### CODE REVIEW

File: addons/sourcemod/scripting/surftimer/misc.sp [hunk 1]
```diff
@@ -1,11 +1,16 @@
-public void SetBotQuota()
+void setBotQuota()
```
Comment (DaLLuZZ) [inline addons/sourcemod/scripting/surftimer/misc.sp]: Don't change the name of the function, please

**Review by DaLLuZZ — CHANGES_REQUESTED** [review]

File: addons/sourcemod/scripting/surftimer/misc.sp (hunk 1 above)
Comment (Nubbeezez) [reply addons/sourcemod/scripting/surftimer/misc.sp]: I did just change it back :) sorry

Comment (DaLLuZZ) [reply addons/sourcemod/scripting/surftimer/misc.sp]:
```cpp
public void SetBotQuota()
```
instead of
```cpp
public void setBotQuota()
```
please

**Comment by Nubbeezez** [comment]
So if im understanding this correctly ? its the maps fault that its getting spammed after the latest riptide update ?

**Comment by Bara** [comment]
> removing FCVAR_REPLICATED is not the best way to solve the problem

Yes, but instead of updating every map (whenever the mapper is still active) it's a worth to add this workaround. Just depends on if it's a map issue.

**Review by 1zc — APPROVED** [review]
As concluded on Discord, there is no real need for bot_quota to be replicated to the client and there is no harm in preventing it from happening. LGTM!

**Pull request merged**